presto-main/src/test/java/io/prestosql/sql/query/TestColumnMask.java

```diff
@@ -351,4 +351,19 @@ public void testInvalidMasks()
             assertions.assertFails("SELECT orderkey FROM orders", "\\Qline 1:20: Column mask for 'orders.orderkey' cannot contain aggregations, window functions or grouping operations: [GROUPING (orderkey)]\\E");
         });
     }
+
+    @Test
+    public void testShowStats()
+    {
+        assertions.executeExclusively(() -> {
+            accessControl.reset();
+            accessControl.columnMask(
+                    new QualifiedObjectName(CATALOG, "tiny", "orders"),
+                    "orderkey",
+                    USER,
+                    new ViewExpression(USER, Optional.of(CATALOG), Optional.of("tiny"), "7"));
+
+            assertions.assertFails("SHOW STATS FOR (SELECT * FROM orders)", "\\QSHOW STATS for table with column masking is not supported: orderkey");
```

Comment:
add

- case when masked column is explicitly selected (alone, or with others)
- case when masked column is not selected